scss/_patterns_navigation.scss

```diff
@@ -651,21 +657,214 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
   }
 
+  @keyframes vf-nav-fade-in {
+    from {
+      opacity: 0;
+    }
+    to {
+      opacity: 0.5;
+    }
+  }
+
+  $sliding-nav-animation-settings: map-get($animation-duration, brisk) ease;
+
+  .p-navigation--sliding.has-menu-open {
+    height: 100vh;
+    overflow-y: hidden;
+
+    @media (min-width: $breakpoint-navigation-threshold) {
+      height: auto;
+      overflow-y: visible;
+    }
+
+    .p-navigation__banner .p-navigation__items {
+      display: flex;
+      flex-wrap: nowrap;
+
+      @media (min-width: $breakpoint-navigation-threshold) {
+        display: none;
+      }
+    }
+    .p-navigation__nav {
+      display: block;
+      height: 100vh;
+      overflow-x: hidden;
+
+      .p-navigation__items {
+        display: block;
+      }
+
+      @media (min-width: $breakpoint-navigation-threshold) {
+        display: flex;
+        height: auto;
+        overflow-x: visible;
+
+        .p-navigation__items {
+          display: flex;
+        }
+      }
+    }
+
+    .p-navigation__items:not(:first-child)::before {
+      content: none;
+    }
+  }
+
+  /* stylelint-disable max-nesting-depth */
+  .p-navigation--sliding .p-navigation__items {
+    transition: transform $sliding-nav-animation-settings;
+
+    &.is-active {
+      transform: translateX(-25vw);
+      @media (min-width: $breakpoint-navigation-threshold) {
+        transform: none;
+      }
+      .p-navigation__dropdown {
+        transform: translateX(25vw);
+        @media (min-width: $breakpoint-navigation-threshold) {
+          transform: none;
+        }
+        &[aria-hidden='false'] {
```

Comment:
This confuses me a bit. Would that work if `aria-hidden` is not on element? 
It seems it would be more natural to have "standard" (visible) styles as default, and `&[aria-hidden='true']` overriding them. Would that work?

Reply:
I guess the idea here is that `aria-hidden=true` is basically the default for every item and then occasionally one of them at a time will be `aria-hidden=false` hence why the override is targetting that

Reply:
The issue with the `[aria-hidden='false']` is that it applies ONLY when aria-hidden is explicitly set to true. If you have no `aria-hidden` on element at all, it means the same (element is not hidden) but the styles won't apply.

Maybe something like `&[aria-hidden='false'], &:not([aria-hidden])` would work? But it shows that this is double negative.
I don't know if it would complicate things to reverse the logic here, but it may make the selector easier, more clear and less reliant on specific JS implementation.

Reply:
Having both: `&[aria-hidden='false'], &:not([aria-hidden])` seems to work in both cases (with aria-hidden false, and without aria-hidden), so I will be fine with that.

Reply:
Let's go with that for now then and see if there's a better way forward